### ISSUE TITLE
Audio file alt text button not visible

### ISSUE THREAD

**Issue by FediVideos** [mastodon/mastodon#40504]
### Steps to reproduce the problem

1. Upload an audio file in the web interface
2. Add an alt text in the upload process
3. Post it, observe that there is no way to view the alt text
...

### Expected behaviour

There should be an ALT button to display the description

### Actual behaviour

There is no ALT button

### Detailed description

Others on other instances have had similar problems.

### Mastodon instance

social.growyourown.services

### Mastodon version

v4.7.1

### Browser name and version

Firefox 155.0.1

### Operating system

Linux Mint 22.3
